src/components/calcite-pick-list/shared-list-logic.ts

```diff
@@ -0,0 +1,135 @@
+import { CalcitePickList } from "./calcite-pick-list";
+import { CalciteValueList } from "../calcite-value-list/calcite-value-list";
+
+function updateSelectedValuesMap(
+  selectedValuesMap: Map<string, HTMLCalcitePickListItemElement> | Map<string, HTMLCalciteValueListItemElement>,
+  item: HTMLCalcitePickListItemElement | HTMLCalciteValueListItemElement
+): void {
+  const selectedValues = (selectedValuesMap as any) as Map<
```

Comment:
Question: Anyway to avoid the `any` and `as` used here?

Reply:
https://github.com/Esri/calcite-app-components/issues/492